src/iuse_actor.cpp

```diff
@@ -3322,6 +3322,11 @@ std::string repair_item_actor::action_description( repair_item_actor::repair_typ
 }
 
 std::string repair_item_actor::get_name() const
+{
+    return string_format( _( "Repair" ) );
```

Comment:
```suggestion
    return _( "Repair" );
```

Reply:
Put the string_format because cast_spell_actor::get_name had it, thought it was an idiomatic way to write it, i'll resolve soon
https://github.com/CleverRaven/Cataclysm-DDA/blob/f0b7411fe50efa396fba91de33e870384890be92/src/iuse_actor.cpp#L2509-L2516